README.md

```
@@ -4,17 +4,220 @@ Decimal floating point math implemented in Solidity/Yul.
+
+#### Unimplemented math
+
+There are some situations where certain operations on some numeric ranges has
+not been implemented and here we error.
+
+Most notable is that we do not currently support powers of negative number bases
+as this often results in complex numbers rather than real numbers.
+
+[rainlanguage/rain.math.float#88](https://github.com/rainlanguage/rain.math.float/issues/88)
+
+#### Parsing/formatting issues
```

markdownlint-cli2 (0.18.1)

Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)